### PR TITLE
PHP Documentor

### PR DESCRIPTION
**Pull request by Endermanbugzjfc**
- Generate documents on merge to Poggit branch / workflow dispatch
- Create commit ~~with GPG signing~~ using the name of Github Action and push to gh-pages branch
- Display in workflow annotation if gh-pages branch does not exist

### CODE REVIEW

**Comment by Endermanbugzjfc** [comment]
# [Preview (not permanent)](https://endermanbugzjfc.github.io/ExamplePlugin/docs)

File: .github/workflows/poggit.yml [hunk 1]
```diff
@@ -91,3 +91,67 @@ jobs:
...
+      run: composer install --prefer-dist --no-interaction
+    - name: Define link param under pull request event context
+      if: contains(github.event_name, 'pull_request')
+      id: link
+      run: |
+        echo "Event name: ${{ github.event_name }}"
+        echo "::set-output name=link::#${{ github.event.pull-request-number }}"
+    - name: Run phpdoc
+      run: |
+        docker run --rm -v $(pwd):/data phpdoc/phpdoc:3 -t ./docs
```
Comment (keopiwauyu) [inline .github/workflows/poggit.yml:148]:
```suggestion
    steps:
    - uses: actions/checkout@v3
      with:
        ref: gh-pages
    - name: Define link param under pull request event context
      if: contains(github.event_name, 'pull_request')
      id: link
      run: |
        echo "Event name: ${{ github.event_name }}"
        echo "::set-output name=link::#${{ github.event.pull-request-number }}"
    - name: Run phpdoc
      run: |
        docker run --rm -v $(pwd):/data phpdoc/phpdoc:3 -t ./docs
```

**Review by keopiwauyu — CHANGES_REQUESTED** [review]
setup php need time nad pcoektmei n have many file

**Comment by keopiwauyu** [comment]
stop put bad word in commit or i ban you

**Pull request merged**